decidim-core/app/views/decidim/open_data/index.html.erb

```diff
@@ -70,6 +70,20 @@
           </div>
         <% end %>
 
+        <hr class="mt-8 mb-4">
+
+        <h3><%= t("core", scope: "decidim.open_data.index.download") %></h3>
```

Comment:
Oh I see that we have forgotten actually testing for these help section in this page. Can you add some specs for some random explanations on `decidim-core/spec/system/download_open_data_files_spec.rb` 🙏🏽 ?